applications/openshift/api-server/api_server_no_adm_ctrl_plugins_disabled/rule.yml

```diff
@@ -0,0 +1,45 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Check that no admission control plugins are disabled'
+
+description: |-
+    To make sure none of them is explicitly enabled, call
+    <pre>$ oc -n openshift-kube-apiserver get configmap config -o json | jq -r '.data."config.yaml"' | jq '.apiServerArguments."disable-admission-plugins"'</pre>
```

Comment:
Are they supposed to be enabled or disabled?

Reply:
oops :)